src/app/users-page/book/add-books/add-books.component.ts

```diff
@@ -103,12 +99,21 @@ export class AddBooksComponent implements OnInit {
   addBook() {
     this.book = this.registerBookForm.value;
     this.book.author = this.authorsList.find((x) => x.id === this.author.value);
-    this.selectedTags = [this.tagsList.find((x) => x.id === this.tags.value)];
-    this.book.tags = this.selectedTags;
+
+    for (let index = 0; index <= this.tags.value.length - 1; index++) {
```

Comment:
Please change this implementation so that it allows one book to have multiple tags.